packages/webapp/pages/onboarding.tsx

```diff
@@ -224,6 +232,15 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.Sources);
     }
 
+    const isSourcesStep = activeScreen === OnboardingStep.Sources;
+    const isLastStepBeforePlus = [
+      OnboardingStep.ContentTypes,
+      OnboardingStep.ReadingReminder,
+    ].includes(activeScreen);
```

Comment:
You can actually just add Sources here right?
If either of those 3 in this point of the checks it should add it?

We use early returns so that should be safe I think?

Reply:
Yes, added here! 💯